storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -1023,6 +1025,14 @@ struct my_option xb_client_options[] =
     "privileges before performing any query.", &opt_check_privileges,
    &opt_check_privileges, 0, GET_BOOL, NO_ARG, 0, 0, 0, 0, 0, 0},
 
+  {"read_buffer_size",
+   OPT_XTRA_DATAFILE_READ_BUFFER_MULTIPLICATOR,
+   "Set datafile read buffer size, given value is scaled up to page size."
+   " Use 0 for default, which is 10Mb.",
```

Comment:
The whole point of having default value of `0` was to be able distinguish user set value from default one. In my opinion, having specific value set as default value would complicate things a bit:
* that would create unnecessary dependency of option-specifying code to the engine-stuff
* default value may change in future engine versions (or different engines may have different default values)
* it is hard to tell if specific value was explicitly set by the user or was that a default one.